### PR TITLE
Decode transactions claiming SAFE tokens from vesting.

### PR DESCRIPTION
**Pull request by LefterisJP**
Fix https://github.com/orgs/rotki/projects/11/views/2?pane=issue&itemId=70040930

### CODE REVIEW

File: rotkehlchen/chain/ethereum/modules/safe/constants.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from typing import Final
+
+from rotkehlchen.chain.evm.types import string_to_evm_address
+
+CPT_SAFE: Final = 'safe'
```
Comment (yabirgb) [inline rotkehlchen/chain/ethereum/modules/safe/constants.py:5]: we already have `CPT_SAFE_MULTISIG`. Does it make sense to keep only `safe` and not have multiple safe things?

File: rotkehlchen/chain/ethereum/modules/safe/decoder.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                to_address=SAFE_VESTING,
+            )],
+        )
+
+    # -- DecoderInterface methods
+
+    def addresses_to_decoders(self) -> dict[ChecksumEvmAddress, tuple[Any, ...]]:
+        return {
+            SAFE_VESTING: (self._decode_safe_vesting,),
+        }
```
Comment (yabirgb) [inline rotkehlchen/chain/ethereum/modules/safe/decoder.py:53]: nitpick: fits in one line

File: rotkehlchen/chain/ethereum/modules/safe/constants.py (hunk 1 above)
Comment (LefterisJP) [reply rotkehlchen/chain/ethereum/modules/safe/constants.py:5]: It's not the same protocol. One is the multisig. The other is the token. Not sure if it makes sense

File: rotkehlchen/chain/ethereum/modules/safe/decoder.py (hunk 2 above)
Comment (LefterisJP) [reply rotkehlchen/chain/ethereum/modules/safe/decoder.py]: done

**Review by yabirgb — APPROVED** [review]
lgtm

**Pull request merged**